pytorch_lightning/tuner/lr_finder.py

```diff
@@ -98,6 +101,9 @@ def lr_find(
             loss at any point is larger than early_stop_threshold*best_loss
             then the search is stopped. To disable, set to None.
 
+        datamodule: A :class:`LightningDataModule`.
```

Comment:
this docstring is not saying anything meaningful. Instead of the class (which is derived from type annotation in docs) could you maybe add a short description here what this parameter does?

Reply:
Sorry for the delay. Added a brief description, although it's rather self-explanatory because I couldn't find any matching documentation on ``trainer.fit()`` to reference.